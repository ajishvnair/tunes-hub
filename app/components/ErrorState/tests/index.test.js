import React from 'react';
import { renderProvider } from '@utils/testUtils';
import { ErrorState } from '../index';

describe('<ErrorState /> tests', () => {
    const keyword = 'test';

    it('should render and match the snapshot', () => {
        const { baseElement } = renderProvider(<ErrorState keyword={keyword} />);
        expect(baseElement).toMatchSnapshot();
    });
});
