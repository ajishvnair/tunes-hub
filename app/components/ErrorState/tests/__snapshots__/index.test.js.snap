// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorState /> tests should render and match the snapshot 1`] = `
<body>
  <div>
    <div
      class="ErrorState__Container-sc-1jjr6tr-0 bgvOzz"
    >
      <p
        class="T__StyledText-gjlic1-0 cbAvsY"
        data-testid="t"
      >
        !Oops there is no tunes for test 
      </p>
    </div>
  </div>
</body>
`;
